<launch>
    <arg name="arm_mode" default="false"/>
  
    <node name="offboard_node" pkg="offboard" type="offboard_node" output="screen">
        <param name="arm_mode_enable" type="bool" value="$(arg arm_mode)"/>

    </node>
</launch>
